pkg/cloud/provider.go

```diff
@@ -266,6 +265,7 @@ func CustomPricesEnabled(p Provider) bool {
 	if err != nil {
 		return false
 	}
+	// TODO:CLEANUP what is going on with this?
```

Comment:
I think we may have had some NaNs parsing this without setting this to 0%, but not sure.

Reply:
Yep, agreed that seems likely. Just noting that we will probably want to be handling that in a place other than the "CustomPricesEnabled" function. For a clean up sometime!